drop-in/src/main/java/com/adyen/checkout/dropin/ComponentParsingProvider.kt

```diff
@@ -333,20 +335,21 @@ internal fun getPaymentMethodAvailabilityCheck(
  * @param storedPaymentMethod The stored payment method to be parsed.
  * @throws CheckoutException In case a component cannot be created.
  */
-@Suppress("UNCHECKED_CAST")
+@Suppress("UNCHECKED_CAST", "LongParameterList")
 internal fun getComponentFor(
     fragment: Fragment,
     storedPaymentMethod: StoredPaymentMethod,
     dropInConfiguration: DropInConfiguration,
     amount: Amount,
     componentCallback: ComponentCallback<*>,
+    sessionSetupConfiguration: SessionSetupConfiguration? = null,
```

Comment:
Let's remove the default value here so we don't forget to pass `sessionSetupConfiguration` whenever this method is used